From 6911d15f6cb327dc15f725da8b9c0e60fbd0be45 Mon Sep 17 00:00:00 2001
From: Larisa Grigore <larisa.grigore@nxp.com>
Date: Fri, 11 Jun 2021 15:57:21 +0300
Subject: [PATCH 1/7] s32: Enable workaround for ARM errata 1530924

Enable workaround for ARM errata 1530924 (NXP errata ERR050764).

Upstream-Status: Pending

Issue: ALB-7186
Signed-off-by: Larisa Grigore <larisa.grigore@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 plat/nxp/s32g/platform.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/plat/nxp/s32g/platform.mk b/plat/nxp/s32g/platform.mk
index ac9761ae0..b8f993699 100644
--- a/plat/nxp/s32g/platform.mk
+++ b/plat/nxp/s32g/platform.mk
@@ -171,6 +171,8 @@ RESET_TO_BL31			:= 0
 # secure GIC registers, which U-Boot and then Linux won't be able to.
 COLD_BOOT_SINGLE_CPU		:= 0
 
+ERRATA_SPECULATIVE_AT	:= 1
+
 ### Platform-specific defines ###
 # Which LinFlexD to use as a UART device
 S32G_LINFLEX_MODULE	:= 0
-- 
2.17.1

